optuna_dashboard/_cached_extra_study_property.py

```diff
@@ -42,6 +43,7 @@ def __init__(self) -> None:
         self._cursor: int = -1
         self._intersection: Optional[SearchSpaceSetT] = None
         self._union: SearchSpaceSetT = set()
```

Comment:
Adding a comment in response to @c-bata san's comment
```suggestion
        # TODO: intersection_search_space and union_search_space look more clear since now we have union_user_attrs.
        self._intersection: Optional[SearchSpaceSetT] = None
        self._union: SearchSpaceSetT = set()
```